pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -143,6 +143,8 @@ func commonArgs(cfg *config.Cluster, networkName string) ([]string, error) {
 		"--net", networkName, // attach to its own network
 		// label the node with the cluster ID
 		"--label", fmt.Sprintf("%s=%s", clusterLabelKey, cfg.Name),
+		// specify container implementation to systemd
+		"-e", "container=podman",
```

Comment:
I couldn't find this documented, what is the use of this environment variable `containre`?

Reply:
systemd, https://systemd.io/CONTAINER_INTERFACE/
we set `container=docker` in the dockerfile, which is a fine fallback (we're just required to set _something_ descriptive), but it makes sense to override this in podman so if you debug this it shows podman instead of docker.

as mentioned in the PR body, the value isn't actually important.

Reply:
TIL, it also fixes the macine-uuid thing